db/app.js

```diff
@@ -1,13 +1,16 @@
 const exp = require("express")
 const app = exp()
-const {getApi} = require("./controller")
+const {getApi, getApiTopics} = require("./controller")
 module.exports = app
 
-app.get("/api", getApi)
+app.use(exp.json())
```

Comment:
Has this line of code been driven by a test? Every code written should be driven by a test or part of a test but we haven't yet got one that requires express.json() yet and therefore this code shouldn't exist at this point! :eyes: